src/backend/assets/js/session.js

```diff
@@ -1,20 +1,21 @@
 $(document).ready(function() {
+  // name is variable defined in the script in the session template file
+  var eventName = name;
```

Comment:
`name` should be in the local scope. How is this accessible here. It should be undefined here.